package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "node-emoji",
-  "version": "1.10.0",
-  "description": "simple emoji support for node.js projects",
+  "version": "2.0.0",
+  "description": "Simple emoji support for Node.js.",
+  "repository": "https://github.com/omnidan/node-emoji.git",
   "author": "Daniel Bugl <daniel.bugl@touchlay.com>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/omnidan/node-emoji.git"
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "engines": {
+    "node": ">=10"
```

Comment:
```suggestion
    "node": ">=14"
```

Reply:
any reason why we should support 14 and higher only?

Reply:
Node 12 is now only in maintenance, and 14 is LTS. I suppose 12 would work too 🙂 but newer Node versions mean we can go with newer JS syntax.

Reply:
I say better keep it on 10 unless it's really needed, or the new additions are just great enough to justify the compatibility breakage.